vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt

```diff
@@ -128,14 +146,6 @@ class WidgetFragment @Inject constructor(
         }
     }
 
-    override fun onPause() {
```

Comment:
Do we possibly want to keep this for non-Element-Call widgets?

Reply:
Exactly, it is in my to-do list. Nice catch though, thanks.

Reply:
Done.